README.md

```diff
@@ -99,11 +99,11 @@ See the [getting started](https://meshery.io/#getting-started) section to quickl
         </tr>
         <tr><td class="beta-adapters"></td></tr>
         <tr>
-          <td rowspan="5" class="alpha-adapters">alpha</td>
+          <td rowspan="4" class="alpha-adapters">alpha</td>
```

Comment:
@UdokaVrede given the feedback in https://github.com/layer5io/meshery/pull/1481, I suspect including the same changes in this PR is by accident. 

No problem. You can remove this commit from the branch used for the #1500 issue.